### ISSUE TITLE
Elaboration on variables parsed from the email app.

### ISSUE THREAD

**Issue by DavidtheGoliath** [frikky/Shuffle#598]
**What do all these mean? and could they be simplified??**

https://user-images.githubusercontent.com/31187099/144708931-44cac002-148f-42ef-ba9c-fca65c519e9e.mp4

**Comment by frikky** [comment]
Absolutely! Setting it up as an enhancement, as it has to do with UX. May need a proper explainer of the different ways autocompletion works.
